tiledb/sm/filesystem/posix.cc

```diff
@@ -520,10 +497,10 @@ Status Posix::write(
   if (buffer == nullptr) {
     throw std::invalid_argument("buffer must not be nullptr");
   }
-  if constexpr (SSIZE_MAX < UINT64_MAX) {
-    if (buffer_size > SSIZE_MAX) {
+  if constexpr (SIZE_MAX < UINT64_MAX) {
+    if (buffer_size > SIZE_MAX) {
```

Comment:
Both being (now) unsigned, another physical impossibility, in both if()s [in 64bit environ - do we still do 32bit build]?

Reply:
Hadn't I reverted this? It should be gone now. 😩